Pages/home.vue.js

```diff
@@ -0,0 +1,19 @@
+const Home = {
+    data: function() {
```

Comment:
Is this the function where we can write CRUD code for database? Also is the Home constant use to pass under routes constant as a value of name property, right?

Reply:
No. This is a "Home Component" which will be the first Vue Component when the site is loaded for the first time. And this function is property of Component Data which can be manipulated for the component. This function will contain data variables. CRUD for database will be placed in `methods` property of Vue Component. 

Also, `Home` is the name of component. So, I want to highlight it that's why I capitalized it. We can use `homeComponent` instead if everyone agrees on that.

Reply:
Ah okay, i see. That make sense now!

And yeah, as you follow naming pattern for all the components should make sense.